### PR TITLE
feat!: migrate to use microgen

### PR DESCRIPTION
**Pull request by arithmetic1728**

### CODE REVIEW

File: samples/snippets/requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-google-cloud-secret-manager==1.0.0
+#google-cloud-secret-manager==1.0.0
```
Comment (busunkim96) [inline samples/snippets/requirements.txt:1]: Nox should be installing the library from source - is that not working correctly?

File: samples/snippets/snippets_test.py [hunk 2]
```diff
@@ -52,21 +52,21 @@ def iam_user():
 
 @pytest.fixture()
 def secret(client, project_id):
-    parent = client.project_path(project_id)
+    parent = f"projects/{project_id}"
```
Comment (busunkim96) [inline samples/snippets/snippets_test.py:55]: Could you run `black` over this directory so the single/double quote usage is consistent?

**Review by busunkim96 — APPROVED** [review]

File: samples/snippets/requirements.txt (hunk 1 above)
Comment (arithmetic1728) [reply samples/snippets/requirements.txt]: > Nox should be installing the library from source - is that not working correctly?

Thank you for catching this! I did this for debugging locally and forgot to revert it. Now reverted.

File: samples/snippets/snippets_test.py (hunk 2 above)
Comment (arithmetic1728) [reply samples/snippets/snippets_test.py:55]: done

**Pull request merged**